### DISCUSSION TITLE
不知道可否添加文档类型的布局，就是在一个目录内打开展示所有目录内容的那种

### DISCUSSION THREAD

**Discussion by IshuaWang** [halo-dev/halo#6930] [Ideas]
在有些时候，这种布局在内容输出或者阅读上都非常方便直观，一目了然。不知道halo现在是不是已经能够实现这种，或者说有没有开发这个功能的计划。

**Comment by ruibaby** [comment]
https://www.halo.run/store/apps/app-yffxw
